include/SFML/System/Angle.inl

```diff
@@ -0,0 +1,218 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2007-2022 Laurent Gomila (laurent@sfml-dev.org)
+//
+// This software is provided 'as-is', without any express or implied warranty.
+// In no event will the authors be held liable for any damages arising from the use of this software.
+//
+// Permission is granted to anyone to use this software for any purpose,
+// including commercial applications, and to alter it and redistribute it freely,
+// subject to the following restrictions:
+//
+// 1. The origin of this software must not be misrepresented;
+//    you must not claim that you wrote the original software.
+//    If you use this software in a product, an acknowledgment
+//    in the product documentation would be appreciated but is not required.
+//
+// 2. Altered source versions must be plainly marked as such,
+//    and must not be misrepresented as being the original software.
+//
+// 3. This notice may not be removed or altered from any source distribution.
+//
+////////////////////////////////////////////////////////////
+
+namespace priv
+{
+    constexpr float pi = 3.141592654f;
```

Comment:
I don't know and am fine either way. You just tell me what you want.

Reply:
We also use it here 🙂 although locally

https://github.com/SFML/SFML/blob/871c021a75e3d728e96086f04f7ffdf61ebd83cb/src/SFML/Graphics/CircleShape.cpp#L73-L82

I'm fine to leave this as-is for now. Should we ever make it public, it would become `PascalCase`.

Reply:
I have another local commit that promotes this constant into the public interface of `sf::Angle` and then replaces all redundant definitions with that one. Let me know if you're interested in this:

https://github.com/ChrisThrasher/SFML/commit/4110f22de49438bd2c83ed37d9b3e21ffd5d842b

Reply:
I'd say we discuss that separately (maybe in an issue before a PR). An option is also that we don't need numeric π, but could add something like `Angle::Pi` or `Angle::HalfCircle` or so.

Reply:
Btw, if you mark conversations as resolved but expect answers, there's a high chance I will miss them and won't respond 😉